rrfs-test/IODA/yaml/prepbufr_adpsfc.yaml

```diff
@@ -190,20 +192,20 @@ observations:
           coordinates: "longitude latitude"
           source: variables/longitude
           longName: "Longitude"
-          units: "degrees_east"
+          units: "degree_east"
```

Comment:
Essentially both `degrees_east` or `degree_east` work although the former looks a little bit preferred.
But we would like to see as less changes as possible.

Reply:
Yeah, I like degrees better. I had it that way, but I saw in one it was degree_east and was just going to be consistent with that. I think I'm now realizing that might have been a small inconsistency and most of the files already had degrees_east. I just happened to look at the only file that had it as degree_east. I will revert.